src/modules/iterate.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.